src/Data/Array/Polarized/Pull/Internal.hs

```diff
@@ -61,41 +63,42 @@ make x n = fromFunction (const x) n
 instance Semigroup (Array a) where
   (<>) = append
 
--- A right-fold of a pull array.
+-- | A right-fold of a pull array.
 foldr :: (a #-> b #-> b) -> b #-> Array a #-> b
 foldr f z (Array g n) = go f z g n
   where go :: (_ #-> _ #-> _) -> _ #-> _ -> _ -> _
         go _ z' _ 0 = z'
         go f' z' g' k = go f' (f' (g' (k-1)) z') g' (k-1)
         -- go is strict in its last argument
 
--- | Extract the length of an array, and give back the original array. This
--- is possible since getting the length of an array doesn't count as consuming
--- it.
+-- | Extract the length of an array, and give back the original array.
 findLength :: Array a #-> (Int, Array a)
 findLength (Array f n) = (n, Array f n)
 
--- | A constructor for pull arrays from a function and specified length.
+-- | @fromFunction arrIndexer len@ constructs a pull array given a function
```

Comment:
```suggestion
-- | @'fromFunction' arrIndexer len@ constructs a pull array given a function
```

Adding the quotes makes the name a link. It's not a link which goes far, but I like this convention :slightly_smiling_face: .